scripts/update-showcase.mjs

```diff
@@ -435,6 +435,17 @@ const scraper = new ShowcaseScraper({
 		"https://souto.tk",
 		"https://unwrapped.studio/",
 		"https://gdscyu.com/",
+		"https://ahuja-lab.in/",
+		"https://roudstudio.com/",
+		"https://oengi.com/",
+		"https://kireerik.github.io/refo/",
+		"https://aiaman.web.app/",
```

Comment:
Also astro. It just has a weird customisation that isn't picked up.
![image](https://github.com/withastro/astro.build/assets/64310361/2e492497-d71f-491e-b059-306723301d92)
```suggestion
```